docs/core.md

```diff
@@ -258,6 +258,11 @@ resolves the plugins and presets and proceeds no further. The expectation is
 that callers will take the config's `.options`, manipulate it as they see fit
 and pass it back to Babel again.
 
+This function accepts one additional option in addition to the standard [options](#options): `showIgnoredFiles`.
```

Comment:
Can we make this less ambiguous (maybe with an example)? It's not clear whether the additional option is a separate parameter or a property of the options object.